File: NERP.Admin.Web/ClientApp/src/app/entities/models/pole.ts
import { Point } from '../../common/geo/geometry';
import { BaseNetworkEntity } from './baseNetworkEntity';
import { Installation } from './installation';
import { Owner } from './owner';

export class Pole extends BaseNetworkEntity {
    poleHeight?: number;
    geometry: Point;
    installation?: Installation = new Installation();
    owner?: Owner = new Owner();
}

File: NERP.NetworkEntities/Models/Pole.cs
using NERP.Common.Models;

namespace NERP.NetworkEntities.Models
{
    public class Pole : BaseNetworkEntity
    {
        public Point Geometry { get; set; }
        public double PoleHeight { get; set; }
        public Installation Installation { get; set; }
        public Owner Owner { get; set; }
        public string LoopLength { get; set; }
    }
}
